Detectors/TPC/qc/include/TPCQC/GPUErrorQA.h

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   GPUErrorQA.h
+/// @author Anton Riedel, anton.riedel@cern.ch
+///
+
+#ifndef AliceO2_TPC_QC_GPUERRORQA_H
+#define AliceO2_TPC_QC_GPUERRORQA_H
+
+#include <memory>
+#include <unordered_map>
+#include <gsl/span>
+
+// root includes
+#include "TH1.h"
+
+// o2 includes
+// #include "DataFormatsTPC/Defs.h"
```

Comment:
use forward declaration for TH1
```suggestion
class TH1;
```